server/game/cards/02.1-NTNR/BackWays.js

```diff
@@ -0,0 +1,44 @@
+const ActionCard = require('../../actioncard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class BackWays extends ActionCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Back Ways',
+            playType: ['noon'],
+            target: {
+                activePromptTitle: 'Select your wanted dude to move',
+                cardCondition: {
+                    location: 'play area',
+                    controller: 'current',
+                    wanted: true
+                },
+                cardType: ['dude'],
+                gameAction: 'moveDude'
+            },
+            handler: context => {
+                const erstwhileLocation = context.target.locationCard;
+                this.game.promptForLocation(context.player, {
+                    activePromptTitle: 'Choose destination for ' + context.target.title,
+                    cardCondition: {
+                        location: 'play area',
+                        condition: card => card !== erstwhileLocation

```

Comment:
There has been a refactor to introduce a `equal` function for cards and player. It should be used from now on.

```suggestion
                        condition: card => !card.equals(erstwhileLocation)
```